app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -29,6 +29,7 @@ def show
             param :name, String, :required => true
             param :value, String, :required => true
           end
+          param :effective_user, String, :required => false
```

Comment:
The downside to this is, it's only useful for SSH (or some other provider that needs this setting).  What about taking a hash of provider options? 

Just wondering what happens if you need to specify some playbook-specific setting for Ansible, the hammer --help and apidocs will be a mess and not clear what options are appropriate when.